provider/oauth2/forms.py

```diff
@@ -336,3 +336,10 @@ def clean(self):
 
         data['client'] = client
         return data
+
+
+class ClientCredentialsGrantForm(ScopeMixin, OAuthForm):
+    """
+    Validate a client credentials grant request.
+    """
+    scope = ScopeChoiceField(choices=SCOPE_NAMES, required=False)
```

Comment:
Merging this functionality isn't urgent; but, I also don't want to run the risk of implementing this incorrectly. I am removing this field.